client/src/ui/base/_themes.scss

```diff
@@ -188,6 +188,34 @@
   --ai-help-icon: #{$mdn-color-light-theme-green-50};
   --ai-help-accent-background-color: #{$mdn-color-light-theme-green-50}10;
 
+  --observatory-bg: rgba(242, 242, 245, 1);
+  --observatory-bg-secondary: rgba(255, 255, 255, 1);
+  --observatory-accent: rgba(155, 101, 255, 1);
+  --observatory-border: rgba(228, 228, 246, 1);
+  --observatory-border-accent: rgba(90, 35, 215, 1);
+  --observatory-pass-icon-bg: rgba(229, 250, 230, 1);
+  --observatory-pass-icon-color: rgba(0, 121, 54, 1);
+  --observatory-fail-icon-bg: rgba(250, 229, 229, 1);
+  --observatory-fail-icon-color: rgba(211, 0, 56, 1);
+  --observatory-table-bg: rgba(255, 255, 255, 1);
+  --observatory-color: #000;
+  --observatory-color-secondary: rgba(105, 105, 105, 1);
+  --observatory-table-header-bg: rgba(249, 249, 251, 1);
+  --observatory-alert: #{$mdn-color-light-theme-red-50};
+  --observatory-check: #{$mdn-color-light-theme-green-40};
+  --observatory-grade-a-bg: #cdfae080;
+  --observatory-grade-a-border: #00793680;
+  --observatory-grade-b-bg: #dbf3a980;
+  --observatory-grade-b-border: #53790080;
+  --observatory-grade-c-bg: #f3eca980;
+  --observatory-grade-c-border: #746a0080;
+  --observatory-grade-d-bg: #f3d1a980;
+  --observatory-grade-d-border: #aa480080;
+  --observatory-grade-f-bg: #f8dada80;
+  --observatory-grade-f-border: #aa000080;
+  --observatory-tooltip-color: rgba(255, 255, 255, 1);
+  --observatory-tooltip-color-secondary: rgba(179, 179, 179, 1);
```

Comment:
We should make sure that related colors are together, and consider sorting these alphabetically:

```suggestion
  --observatory-bg: rgba(242, 242, 245, 1);
  --observatory-bg-secondary: rgba(255, 255, 255, 1);
  --observatory-accent: rgba(155, 101, 255, 1);
  --observatory-border: rgba(228, 228, 246, 1);
  --observatory-border-accent: rgba(90, 35, 215, 1);
  --observatory-pass-icon-bg: rgba(229, 250, 230, 1);
  --observatory-pass-icon-color: rgba(0, 121, 54, 1);
  --observatory-fail-icon-bg: rgba(250, 229, 229, 1);
  --observatory-fail-icon-color: rgba(211, 0, 56, 1);
  --observatory-color: #000;
  --observatory-color-secondary: rgba(105, 105, 105, 1);
  --observatory-table-bg: rgba(255, 255, 255, 1);
  --observatory-table-header-bg: rgba(249, 249, 251, 1);
  --observatory-alert: #{$mdn-color-light-theme-red-50};
  --observatory-check: #{$mdn-color-light-theme-green-40};
  --observatory-grade-a-bg: #cdfae080;
  --observatory-grade-a-border: #00793680;
  --observatory-grade-b-bg: #dbf3a980;
  --observatory-grade-b-border: #53790080;
  --observatory-grade-c-bg: #f3eca980;
  --observatory-grade-c-border: #746a0080;
  --observatory-grade-d-bg: #f3d1a980;
  --observatory-grade-d-border: #aa480080;
  --observatory-grade-f-bg: #f8dada80;
  --observatory-grade-f-border: #aa000080;
  --observatory-tooltip-color: rgba(255, 255, 255, 1);
  --observatory-tooltip-color-secondary: rgba(179, 179, 179, 1);
```

Reply:
done